### ISSUE TITLE
AOT compilation fails

### ISSUE THREAD

**Issue by ShankarSumanth** [qdouble/angular-webpack2-starter#272]
Just cloned the git repo and followed the instruction to get an aot build. And received below error

```
ERROR in ./src/main.browser.aot.ts
Module not found: Error: Can't resolve '../compiled/src/app/app.module.ngfactory' in '/Users/sumanthshankar/Projects/angular-webpack2-starter/src'
 @ ./src/main.browser.aot.ts 5:0-78

ERROR in [at-loader] src/main.browser.aot.ts:7:36 
    TS2307: Cannot find module '../compiled/src/app/app.module.ngfactory'.
```
There is somewhere in between this warning as well
```
(node:79540) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: compiler_1.isSyntaxError is not a function
(node:79540) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Also this is the complete build output, if it helps 
```
[at-loader] Checking finished with 1 errors
Hash: 5d03b77d73ef7c19705b
Version: webpack 2.2.0-rc.3
Time: 11437ms
[./node_modules/@angular/core/index.js] ./~/@angular/core/index.js 355 bytes {0} [built]
[./node_modules/@angular/core/src/core.js] ./~/@angular/core/src/core.js 1.76 kB {0} [built]
[./node_modules/@angular/platform-browser/index.js] ./~/@angular/platform-browser/index.js 379 bytes {0} [built]
[./node_modules/@angular/platform-browser/src/platform-browser.js] ./~/@angular/platform-browser/src/platform-browser.js 893 bytes {0} [built]
[./node_modules/@ngrx/core/add/operator/select.js] ./~/@ngrx/core/add/operator/select.js 170 bytes {0} [built]
[./node_modules/core-js/client/shim.js] ./~/core-js/client/shim.js 228 kB {0} [built]
[./node_modules/core-js/es7/reflect.js] ./~/core-js/es7/reflect.js 510 bytes {0} [built]
[./node_modules/rxjs/add/operator/take.js] ./~/rxjs/add/operator/take.js 193 bytes {0} [built]
[./node_modules/rxjs/add/operator/takeUntil.js] ./~/rxjs/add/operator/takeUntil.js 223 bytes {0} [built]
[./node_modules/rxjs/add/operator/toArray.js] ./~/rxjs/add/operator/toArray.js 211 bytes {0} [built]
[./node_modules/ts-helpers/index.js] ./~/ts-helpers/index.js 2.69 kB {0} [built]
[./node_modules/zone.js/dist/zone.js] ./~/zone.js/dist/zone.js 58.2 kB {0} [built]
[./src/main.browser.aot.ts] ./src/main.browser.aot.ts 1.64 kB {0} [built]
[./src/polyfills.browser.aot.ts] ./src/polyfills.browser.aot.ts 113 bytes {0} [built]
[./src/rxjs.imports.ts] ./src/rxjs.imports.ts 916 bytes {0} [built]
    + 320 hidden modules

ERROR in ./src/main.browser.aot.ts
Module not found: Error: Can't resolve '../compiled/src/app/app.module.ngfactory' in '/Users/sumanthshankar/Projects/angular-webpack2-starter/src'
 @ ./src/main.browser.aot.ts 5:0-78

ERROR in [at-loader] src/main.browser.aot.ts:7:36 
    TS2307: Cannot find module '../compiled/src/app/app.module.ngfactory'.
```

The commands used were,
yarn
yarn run compile

**Comment by qdouble** [comment]
@ShankarSumanth which branch?

**Comment by ShankarSumanth** [comment]
@qdouble 

This was the checkout from the master branch I believe.

**Comment by qdouble** [comment]
@ShankarSumanth I didn't update the main branch in a while because Universal was too unstable... I just merged a PR updating it 4.1 though. Let me know if you're still having problems with it.

**Comment by ShankarSumanth** [comment]
@qdouble 👍 . The master branch works now. Thanks for the fix
